GNUmakefile

```diff
@@ -78,10 +56,6 @@ ifeq (,$(wildcard $(GOPATH)/src/$(WEBSITE_REPO)))
 endif
 	@$(MAKE) -C $(GOPATH)/src/$(WEBSITE_REPO) website-provider PROVIDER_PATH=$(shell pwd) PROVIDER_NAME=$(PKG_NAME)
 
-website-lint:
-	@echo "==> Checking website against linters..."
-	@misspell -error -source=text website/
-
```

Comment:
I think we can stick with this one (tested it locally)
It can be helpful to find misspellings in the website documentations
so put it back and invoke `make website-lint` from the travis yaml

Reply:
It's not working, we will add this to our tasks